provider/database.py

```diff
@@ -18,13 +18,19 @@
 from cloudant.result import Result
 
 class Database:
+    dbname = "ow_kafka_triggers"
```

Comment:
@dubeejw I see something similar in the cloudant and alarms trigger deployment scripts. I've ask Jason about it in #34, and we can handle it there if needed.